proto/proto/sighash.proto

```diff
@@ -26,5 +27,8 @@ message SigHashAction {
   oneof action {
     transaction.SpendBody spend = 1;
     transaction.Output output = 2;
+    stake.Delegate delegate = 3;
+    stake.Undelegate undelegate = 4;
+    stake.ValidatorDefinition validator_definition = 16;
```

Comment:
Field numbers are encoded as varints, so the Proto style guide recommends that uncommon variants should be encoded using numbers that don't fit in a single byte.